doc/src/build/move.md

```diff
@@ -767,13 +767,31 @@ At this point, however, the
 `sui-move` command does not support package publishing. In fact, it is
 not clear if it even makes sense to accommodate package publishing,
 which happens once per package creation, in the context of a unit
-testing framework. Instead, one can use a sample Sui wallet to
+testing framework. 
+
+Instead, one can use a sample Sui wallet to
 [publish](wallet.md#publish-packages) Move code and to
 [call](wallet.md#calling-move-code) it. See the
 [wallet documentation](wallet.md) for a description of how
 to publish the package we have [written](#writing-a-package) as as
 part of this tutorial.
 
+Another option is to use the [REST](rest-api.md) endpoints to [publish](rest-api.md#post-publish) 
+the Move code and then [call](rest-api.md#post-call) it. To do this the Sui REST Server requires a hex representation of the compiled modules you wish to publish. The `sui-move` command supports printing the bytecodes as hex with the following option
+
+```
+sui-move --path <move-module-path> build --dump-bytecode-as-hex
+```
+
+You can copy the ouputting hex into the REST publish endpoint. i.e.
+
+```
+sui-move --path sui_programmability/examples/fungible_tokens build --dump-bytecode-as-hex
```

Comment:
We are trying pretty hard not to drop a path like `sui_programmability/examples/fungible_tokens` on the reader out of the blue. Whether this part stays in here or moves to another file, this should be taken into consideration.

In particular, if it stays in move.md, we should probably use a package we are developing here along with the directory structure that is already defined in this document. Again, @Clay-Mysten may have an opinion on this as well.

Reply:
Let's move to rest-api.md.